modules/tensor_mechanics/include/userobjects/LAROMData.h

```diff
@@ -0,0 +1,84 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#pragma once
+
+#include "GeneralUserObject.h"
+
+class LAROMData;
+
+template <>
+InputParameters validParams<LAROMData>();
+
+class LAROMData : public GeneralUserObject
+{
+public:
+  LAROMData(const InputParameters & parameters);
+
+  virtual void initialSetup() override;
+  virtual void initialize() override{};
+  virtual void execute() override {}
+  virtual void finalize() override {}
+
+  /// Calculates and returns the number of inputs for the ROM data set
+  unsigned int getNumberInputs() const;
```

Comment:
I'd call those `getNumberOf...`. It's just two extra chars

Reply:
good idea